src/main/java/de/retest/recheck/testcase/TestFramework.java

```diff
@@ -0,0 +1,7 @@
+package de.retest.recheck.testcase;
+
+public interface TestFramework {
+
+	boolean isTestCase( String annotationName );
```

Comment:
Is this really a proper name for this method? Because we not just look for test cases, but also for setup and teardown methods such as `@Before`.

Reply:
Well ... what we really want to do is extract the name of the test case, to use it for the naming of the resulting `*.check` file. `@Before` and others are only indicators, to help with that.

What name would you suggest?